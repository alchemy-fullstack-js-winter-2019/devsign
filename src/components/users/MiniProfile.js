import React from 'react';

export default function MiniProfile() {
  return (
    <aside>
      <img src="../assets/prof-pic.png" width="30px" alt="prof pic"/>
      <h3>User Name</h3>
      <hr />
      <p>Chirps: 593</p>
      <p>Followers: 388</p>
    </aside>
  );
}
