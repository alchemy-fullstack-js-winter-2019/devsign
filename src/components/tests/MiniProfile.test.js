import React from 'react';
import renderer from 'react-test-renderer';
import MiniProfile from '../users/MiniProfile';

describe('MiniProfile', () => {
  it('matches a snapshot', () => {
    const tree = renderer.create(
      <MiniProfile />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
