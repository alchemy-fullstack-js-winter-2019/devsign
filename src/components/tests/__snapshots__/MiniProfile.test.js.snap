// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MiniProfile matches a snapshot 1`] = `
<aside
  className="sc-bdVaJa qjbAC"
>
  <img
    alt="background"
    className="background"
    src="../assets/background.jpg"
  />
  <div
    className="main"
  >
    <img
      alt="prof pic"
      className="profpic"
      src="../assets/prof-pic.png"
    />
    <div
      className="header"
    >
      <h3>
        Kristin Hortsch
      </h3>
      <p>
        @kristinhortsch
      </p>
    </div>
  </div>
  <table>
    <thead>
      <tr>
        <th>
          Chirps
        </th>
        <th>
          Followers
        </th>
        <th>
          Following
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>
          593
        </td>
        <td>
          388
        </td>
        <td>
          301
        </td>
      </tr>
    </tbody>
  </table>
</aside>
`;
